pkg/kn/commands/broker/broker.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
+
+import (
+	"github.com/spf13/cobra"
+
+	"knative.dev/client/pkg/kn/commands"
+)
+
+func NewBrokerCommand(p *commands.KnParams) *cobra.Command {
+	brokerCmd := &cobra.Command{
+		Use:   "broker",
+		Short: "Manage message broker",
+	}
+	brokerCmd.AddCommand(NewBrokerCreateCommand(p))
+	brokerCmd.AddCommand(NewBrokerDeleteCommand(p))
+	brokerCmd.AddCommand(NewBrokerListCommand(p))
+	return brokerCmd
```

Comment:
The question is whether a `kn broker describe` makes sense. I now a broker does not have many attributes, but at least the status could be helpful for a broker describe. 

Just mentioning that here, being aware that the PR is about create/delete/list

Reply:
There's status and then the most valuable value is generated URL. I'll add describe and with that there'll be use for `Get` method also. :)

Reply:
👍